simtools/model/array_model.py

```diff
@@ -136,43 +147,33 @@ def site(self):
         return self.site_model.site
 
     def _set_config_file_directory(self):
-        """
-        Define and create config file directory.
-
-        """
+        """Define and create config file directory."""
         self._config_file_directory = self.io_handler.get_output_directory(self.label, "model")
 
-    def _build_array_model(self, site, parameters_to_change=None):
+    def _build_array_model(self, site_model, array_elements, parameters_to_change=None):
```

Comment:
maybe you can rename the function and adjust the description here, as it now only returns telescope model and `site_model` is already used as input.

Reply:
Good point, changed it to `_build_telescope_models`.